packages/core/lib/commands/install/run.js

```diff
@@ -1,9 +1,7 @@
-module.exports = async function (options) {
-  const Config = require("@truffle/config");
-  const Package = require("../../package");
-
-  if (options._ && options._.length > 0) options.packages = options._;
-
-  const config = Config.detect(options);
-  return await Package.install(config);
+module.exports = async function () {
```

Comment:
curious about the async here. Is it needed?

Reply:
Thanks for pointing this out @lsqproduction. `async` is not needed now since we have removed the `await` call. I will remove it.